bgpd/bgp_memory.c

```diff
@@ -126,3 +126,8 @@ DEFINE_MTYPE(BGPD, BGP_FLOWSPEC_RULE_STR, "BGP flowspec rule str")
 DEFINE_MTYPE(BGPD, BGP_FLOWSPEC_COMPILED, "BGP flowspec compiled")
 DEFINE_MTYPE(BGPD, BGP_FLOWSPEC_NAME, "BGP flowspec name")
 DEFINE_MTYPE(BGPD, BGP_FLOWSPEC_INDEX, "BGP flowspec index")
+
+DEFINE_MTYPE(BGPD, BGP_LABEL_CHUNK, "BGP Label Chunk")
+DEFINE_MTYPE(BGPD, BGP_LABEL_FIFO, "BGP Label FIFO")
+DEFINE_MTYPE(BGPD, BGP_LABEL_CB, "BGP Dynamic Label Assignment")
+DEFINE_MTYPE(BGPD, BGP_LABEL_CBQ, "BGP Dynamic Label Callback")
```

Comment:
proposal use DEFINE_MTYPE_STATIC to corner the definition only on bgp label pool

Reply:
I think you only have to insert DEFINE_MTYPE_STATIC on top of .c file.
example:
DEFINE_MTYPE_STATIC(BGP, BGP_LABEL_CHUNK, "BGP label chunk")